waveorder/models/inplane_oriented_thick_pol3d.py

```diff
@@ -98,12 +142,9 @@ def apply_inverse_transfer_function(
         *background_corrected_stokes
     )
 
-    # Return retardance in distance units (matching wavelength_illumination)
```

Comment:
Reviewers: this is the "critical" change. 

I'm now skipping the multiplication by `wavelength_illumination /  (2 * np.pi)`